app/models/venue.rb

```diff
@@ -5,6 +5,8 @@ class Venue < ActiveRecord::Base
   validates :zip_code, presence: true, length: { is: 5 }
   validates :zip_code, numericality: { only_integer: true }
   validates :city, presence: true
+  validates :user_id, presence: true
 
```

Comment:
This is fine as is, but it can generally be more convenient to validate `:user` rather than `:user_id`. If you ever get into a situation where you need to create two associated models at once, and they both validate the presence of the other, validations using `:user_id` will fail (since the object hasn't been saved to the database yet), but will pass if just `:user` (as this will validate on presence of an in-memory object, rather than in-database, which allows validations to pass)..

Again, this is perfectly fine here, but it could be something you run into some other time if validating an :id